### ISSUE TITLE
Ubuntu 15.04 can not complete installation using PPA installation procedure

### ISSUE THREAD

**Issue by EdDixon** [learningequality/installers#111]
Having completed the current development PPA installation on 14 LTS, I tested installing on ubuntu 15.04 and ran into the following error before the installation procedure completed but after the assessment items had been downloaded and inflated:

```
Creating config file /etc/init.d/ka-lite with new version
insserv: script ka-lite is not an executable regular file, skipped!
Failed to start ka-lite.service: Unit ka-lite.service failed to load: No such file or directory.
invoke-rc.d: initscript ka-lite, action "start" failed.
dpkg: error processing package ka-lite (--configure):
 subprocess installed post-installation script returned error exit status 6
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by benjaoming** [comment]
@EdDixon Syntax matters, remember a line break after `````.

**Comment by benjaoming** [comment]
( I edited the description )

**Comment by benjaoming** [comment]
Solved, also I made a dupe in #124
